### PR TITLE
chore: modify mtls test to work with provider

### PR DESCRIPTION
**Pull request by danielskinstad**
Tested that it works locally - however `mendersoftware/mender-client-qemu:mender-master` doesn't have the pkcs11-provider

### CODE REVIEW

**Comment by danielskinstad** [comment]
@mender-test-bot start pipeline --pr meta-mender/2167

**Comment by lluiscampos** [comment]
@mender-test-bot start pipeline

**Comment by danielskinstad** [comment]
@mender-test-bot start pipeline

File: tests/tests/test_mtls.py [hunk 1]
```diff
@@ -187,6 +192,8 @@ def common_test_mtls_enterprise(self, env, algorithm=None, use_hsm=False):
         tmpdir = tempfile.mkdtemp()
 
         ssl_engine_id = "pkcs11"
+        if not use_engine:
+            ssl_engine_id = ""
```
Comment (lluiscampos) [inline tests/tests/test_mtls.py:196]: Doesn't the following read simpler?

```suggestion
        ssl_engine_id = ""
        if use_engine:
            ssl_engine_id = "pkcs11"
```

**Review by lluiscampos — APPROVED** [review]
Looks good to me.

File: tests/tests/test_mtls.py (hunk 1 above)
Comment (danielskinstad) [reply tests/tests/test_mtls.py]: It does indeed

**Comment by danielskinstad** [comment]
@mender-test-bot start pipeline

**Pull request merged**